src/TraceEvent/Computers/StartStopActivityComputer.cs

```diff
@@ -74,8 +74,8 @@ public StartStopActivityComputer(TraceLogEventSource source, ActivityComputer ta
 
             Dictionary<long, int> mapConnectionToTargetProcess = new Dictionary<long, int>();
 #endif 
-            var dynamicParser = source.Dynamic;
-            dynamicParser.All += delegate (TraceEvent data)
+            //var dynamicParser = source.Dynamic;
```

Comment:
I want to enable DynamicParser to handle EventSource events from EventPipe. Similar to RegisteredTraceEventParser, I add EventPipeTraceEventParser to DynamicTraceEventParser. It seems working. Can you take a look at the latest change?